app/controllers/application_controller.rb

```diff
@@ -1,4 +1,16 @@
 # frozen_string_literal: true
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+
+  def append_info_to_payload(payload)
```

Comment:
But even if Decidim's controller inherit from the ApplicationController we couldnt add this code to the ApplicationController since there's no current_organization method (unless we add it and it returns nil). I don't think it would be bad to add this to Decidim's controller.